fx_crash_sig/crash_processor.py

```diff
@@ -5,12 +5,32 @@
 from __future__ import print_function
 
 import json
+import pkg_resources
+import sys
+from siggen import (
+    __version__ as siggen_version,
+    __releasedate__ as siggen_releasedate
+)
 from siggen.generator import SignatureGenerator
 
 from fx_crash_sig import SYMBOLS_API
 from fx_crash_sig.symbolicate import Symbolicator
 
 
+def get_version_info():
```

Comment:
Should `get_version_info` be in `crash_processor.py`?  I think it would make more sense to just be in `cmd_get_crash_sig.py` or `__init__.py`.  If you don't agree you can just leave it.

Reply:
It's better if it's in the library rather than just in the command. Then if someone uses fx-crash-sig like a library and not like a command, it's still available. So I think it shouldn't be in `cmd_get_crash_sig.py`.

I'll move it to `__init__.py`.